ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.ts

```diff
@@ -0,0 +1,488 @@
+          return { infoText: output.busterOnYou!() };
+        }
+
+        if (data.role !== 'tank' && data.role !== 'healer')
+          return;
+
+        return { infoText: output.busterOnTarget!({ player: data.ShortName(matches.target) }) };
+      },
+    },
+    {
+      id: 'ASS Total Wash',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7750', source: 'Silkie', capture: false }),
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'aoe + bleed',
+          de: 'AoE + Blutung',
+          fr: 'AoE + Saignement',
+          ja: 'AOE + 出血',
+          ko: '전체 공격 + 도트',
+        },
+      },
+    },
+    // ---------------- Gladiator of Sil'dih ----------------
+    {
+      id: 'ASS Flash of Steel',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: '7671', source: 'Gladiator of Sil\'dih', capture: false }),
+      response: Responses.aoe(),
+    },
+    {
+      id: 'ASS Rush of Might Collect',
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['765C', '765B'], source: 'Gladiator of Sil\'dih' }),
+      preRun: (data, matches) => data.mightCasts.push(matches),
+    },
+    {
+      id: 'ASS Rush of Might 1',
+      // Boss casts 765C (12.2s) and 765B (10.2s), twice
+      // Gladiator of Mirage casts 7659, 7658, 765A, these target the environment but are not reliable
+      // North
+      //                East               West
+      //   Line 1: (-34.14, -270.14) (-35.86, -270.14)
+      //   Line 2: (-39.45, -275.45) (-30.55, -275.45)
+      //   Line 3: (-44.75, -280.75) (-25.25, -280.75)
+      // South
+      //                East               West
+      //   Line 1: (-34.14, -271.86) (-35.86, -271.86)
+      //   Line 2: (-39.45, -266.55) (-30.55, -266.55)
+      //   Line 3: (-44.75, -261.25) (-25.25, -261.25)
+      // Center is at (-35, -271)
+      type: 'StartsUsing',
+      netRegex: NetRegexes.startsUsing({ id: ['765C', '765B'], source: 'Gladiator of Sil\'dih', capture: false }),
+      delaySeconds: 0.1,
+      suppressSeconds: 1,
+      infoText: (data, _matches, output) => {
+        if (data.mightCasts.length !== 4)
+          return;
+
+        const mirage1 = data.mightCasts[0];
+        const unfilteredMirage2 = data.mightCasts[1];
+        const unfilteredMirage3 = data.mightCasts[2];
+
+        if (mirage1 === undefined || unfilteredMirage2 === undefined || unfilteredMirage3 === undefined)
+          throw new UnreachableCode();
+
+        // Filter for unique
+        const mirage2 = (mirage1.x === unfilteredMirage2.x && mirage1.y === unfilteredMirage2.y) ? unfilteredMirage3 : unfilteredMirage2;
+
+        const x1 = parseFloat(mirage1.x);
+        const y1 = parseFloat(mirage1.y);
+        const x2 = parseFloat(mirage2.x);
+        const y2 = parseFloat(mirage2.y);
+
+        const getLine = (x: number) => {
+          if ((x > -46 && x < -43) || (x > -27 && x < -24))
```

Comment:
FWIW, you could do something like `Math.round(x / 5) * 5` if you wanted to round to the nearest multiple of 5.  That might make a lot of this easier to read.

Reply:
I'll modify this later.

Reply:
Using 5 works for x value. For y value, highest integer was 3 that seems to work given the exact floating point values for the positions.